smurfs/src/actions/index.js

```diff
@@ -13,3 +13,40 @@
    U - updateSmurf
    D - deleteSmurf
 */
+import axios from 'axios';
+
+export const FETCHING = 'FETCHING';
+export const SUCCESS = 'SUCCESS' ;
```

Comment:
Consider using a DOUBLE_NAMED constant like FETCHING_SMURFS to be more clear when you get bigger applications